qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/homepage/name_field_validation/name_validation.feature

```diff
@@ -0,0 +1,99 @@
+Feature: 1.1.2. Name field  
+Validate name field  in job position 
+
+
+
+@positive
+Scenario Outline: 1.1.2 Validation with latin characters
+	Given User clicked on 
+	When user clicks on <Name_field> Text field
+	And User fills <Name_field> with latin characters
+		|Name_field				|
+		|John					|
+		|John Simons			|
+		|John Simons Andrew		|
+		|Isabella a Bella       |
+
+	And user navigates away from blurs the <name_field>
+	Then The system displayed that <name_field> field correct
+
+@Positive
+	Scenario Outline: 1.1.3. Validation with kirilica characters
+	Given User in job position page
+	When User fills <Name_field> with kirilica characters
```

Comment:
I'm not sure it's actually cyrillic... Maybe you should rename this scenario.

Reply:
Github shows cyrillic characters in this way

Reply:
ok!